### PR TITLE
ci(cypress): Added 3DS Refund flow 

### PR DESCRIPTION
**Pull request by preetamrevankar**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
Added three_ds Refund flows in the cypress automation for existing connectors

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?
Cypress test 
cybersource connector
Screenshot 2024-05-13 at 21 04 40

Stripe test 
Screenshot 2024-05-13 at 22 25 29

### Trustpay
Screenshot 2024-05-14 at 20 17 11

### Nmi 
Screenshot 2024-05-14 at 20 27 50
same tests are failed in the main

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js [hunk 1]
```diff
@@ -342,3 +342,331 @@ describe("Card - Refund flow test", () => {
...
+            cy.createPaymentIntentTest(createPaymentBody, det, "three_ds", "manual", globalState);
+        });
+
+        it("payment_methods-call-test", () => {
+            cy.paymentMethodsCallTest(globalState);
+        });
+
+
+        it("confirm-call-test", () => {
+            console.log("confirm -> " + globalState.get("connectorId"));
```
Comment (Gnanasundari24) [inline cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js:487]: Remove this unwanted log from all places

File: cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js [hunk 2]
```diff
@@ -342,3 +342,331 @@ describe("Card - Refund flow test", () => {
...
+            let det = getConnectorDetails(globalState.get("connectorId"))["card_pm"]["3DS"];
+            console.log("det -> " + det.card);
```
Comment (Gnanasundari24) [inline cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js:489]: This log also needs to be removed

**Review by Gnanasundari24 — APPROVED** [review]

**Review by likhinbopanna — APPROVED** [review]

File: cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js (hunk 1 above)
Comment (preetamrevankar) [reply cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js]: Done.

File: cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js (hunk 2 above)
Comment (preetamrevankar) [reply cypress-tests/cypress/e2e/ConnectorTest/00008-RefundPayment.cy.js]: Done.

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**